### PR TITLE
Feature 17 landing/home page

### PR DESCRIPTION
**Pull request by Toinne**
TODO:

- [x] Update 2nd Story component to small version

Screenshot 2020-12-27 at 15 46 42

### CODE REVIEW

**Review by Sayed94h — APPROVED** [review]
Hey there 
I checked this PR and finished the second component.
So, I am going to approve this PR.
Have a great day

File: client/package.json [hunk 1]
```diff
@@ -1,37 +1,40 @@
...
-      "not op_mini all"
-    ],
-    "development": [
-      "last 1 chrome version",
-      "last 1 firefox version",
-      "last 1 safari version"
-    ]
-  },
-  "proxy": "http://localhost:5000"
+	"name": "phht",
```
Comment (abelRoland) [inline client/package.json:2]: Is it not nice to put the name of the project "Hear my story" or something like this?

File: client/src/Gallery.css [hunk 2]
```diff
@@ -0,0 +1,12 @@
+section.Gallery {
+	background: url("./children.jpg");
+	background-size: contain;
+	background-repeat: no-repeat;
+	background-position-x: center;
+	background-position-y: 4em;
+	width: 100%;
+
```
Comment (abelRoland) [inline client/src/Gallery.css:8]: No need of extra line?

**Review by abelRoland — COMMENTED** [review]
Everything seems very nice.
I am just not sure if to make it in classes components instead of functional components was an architectural decision of you guys, but you guys rules... Great job!

File: client/package.json (hunk 1 above)
Comment (Sayed94h) [reply client/package.json:2]: I agree

**Review by yaredyilma — APPROVED** [review]

**Review by murvanessa — APPROVED** [review]

**Pull request merged**